src/main/java/org/apache/maven/archiver/MavenArchiver.java

```diff
@@ -615,6 +620,9 @@ public void createArchive( MavenSession session, MavenProject project,
         Manifest manifest = getManifest( session, workingProject, archiveConfiguration );
 
         // Configure the jar
+
+        archiver.setMinimalDefaultManifest( true );
```

Comment:
This change  seems unrelated

Reply:
Thanks for pointing that out. This belongs into another ticket. I will drop this hunk.

Reply:
you are welcome !
cheers